AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.kt

```
@@ -105,7 +105,18 @@ open class RobolectricTest : CollectionGetter {
             RustBackendLoader.loadRsdroid(backendPath)
         } else {
             // default (no env variable): Extract the backend testing lib from the jar
-            RustBackendLoader.init()
+            try {
+                RustBackendLoader.init()
+            } catch (e: UnsatisfiedLinkError) {
+                if (e.message.contains("arm64e")) {
```

If the message doesn't contain `arm64e`, we want to rethrow the exception